src/callstack.cpp

```diff
@@ -383,7 +383,7 @@ int CallStack::resolve(BOOL showInternalFrames)
     IMAGEHLP_LINE64  sourceInfo = { 0 };
     sourceInfo.SizeOfStruct = sizeof(IMAGEHLP_LINE64);
 
-    bool skipStartupLeaks = !!(g_vld.GetOptions() & VLD_OPT_SKIP_CRTSTARTUP_LEAKS);
+    //bool skipStartupLeaks = !!(g_vld.GetOptions() & VLD_OPT_SKIP_CRTSTARTUP_LEAKS);
```

Comment:
Please do not leave dead code as comments.

Reply:
right, sorry